This removes a bunch of parsing of /proc/cpuinfo and the
likes since that's not reproducible and broken for packaging.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,6 @@ SET(CMAKE_BUILTTYPE None)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/CMakeModules/")
 
-include (FindSSE)
-
 set(MAJOR_VERSION 1)
 set(MINOR_VERSION 1)
 set(PATCH_VERSION 0)
